roles/openshift_openstack/templates/heat_stack_server.yaml.j2

```diff
@@ -212,6 +212,9 @@ resources:
         host-type: { get_param: type }
         sub-host-type:    { get_param: subtype }
         node_labels: { get_param: node_labels }
+{% if openshift_openstack_dns_nameservers %}
```

Comment:
note, the default is [], so this will be always matching. Please additionally check is not empty

Reply:
Empty array evaluates to false so the condition is only true for nonempty arrays. But I can make it more explicit if you want.

Reply:
indeed, LGTM then